scripts/brc4/json_samples_to_redmine_samples.py

```diff
@@ -0,0 +1,66 @@
+#!env python3
+
+import argparse
+import json
+from pathlib import Path
+
+
+class Dataset:
+
+    def __init__(self, ds) -> None:
+        self.component = ds["component"]
+        self.species = ds["species"]
+        self.name = ds["name"]
+        self.samples = ds["runs"]
+
+    def __str__(self) -> str:
+        lines = []
+        for sample in self.samples:
+            lines.append(f"{sample['name']} : {','.join(sample['accessions'])}")
+        return "\n".join(lines)
+
+    def full(self) -> str:
+        lines = []
+        cols_init = (self.component, self.species, self.name)
+        for sample in self.samples:
+            cols = list(cols_init)
+            cols.append(sample['name'])
+            lines.append("\t".join(cols))
```

Comment:
Just a suggestion to avoid some repetitive tasks:
```suggestion
        cols = f"{self.component}\t{self.species}\t{self.name}"
        for sample in self.samples:
            lines.append(f"{cols}\t{sample['name']}")
```

Reply:
I prefer to keep the elements in a list form until they are joined (though I made it a bit shorter/cleaner)